lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -87,7 +114,11 @@ def cmd_run(*args)
       when '-q'
         quiet  = true
       when '-h'
-        cmd_run_help
+        if cmd == 'run'
```

Comment:
Then here, to know if we're running an action or the default run, this should be:
```suggestion
        if action.nil?
```